diff -rNuw Box2D_v2.0.1/Box2D/Source//Collision/b2BroadPhase.cpp Box2D_v2.0.1/Box2Dfixed/Source/Collision/b2BroadPhase.cpp
--- Box2D_v2.0.1/Box2D/Source//Collision/b2BroadPhase.cpp	2008-03-10 21:42:36.000000000 -0700
+++ Box2D_v2.0.1/Box2Dfixed/Source/Collision/b2BroadPhase.cpp	2009-06-24 18:47:08.103361670 -0700
@@ -18,6 +18,7 @@
 
 #include "b2BroadPhase.h"
 #include <algorithm>
+#include <cstring>
 
 // Notes:
 // - we use bound arrays instead of linked lists for cache coherence.
diff -rNuw Box2D_v2.0.1/Box2D/Source//Common/b2BlockAllocator.cpp Box2D_v2.0.1/Box2Dfixed/Source/Common/b2BlockAllocator.cpp
--- Box2D_v2.0.1/Box2D/Source//Common/b2BlockAllocator.cpp	2007-11-24 11:25:02.000000000 -0800
+++ Box2D_v2.0.1/Box2Dfixed/Source/Common/b2BlockAllocator.cpp	2009-06-24 18:46:41.839360729 -0700
@@ -20,6 +20,7 @@
 #include <cstdlib>
 #include <memory>
 #include <climits>
+#include <cstring>
 
 int32 b2BlockAllocator::s_blockSizes[b2_blockSizes] = 
 {
diff -rNuw Box2D_v2.0.1/Box2D/Source//Dynamics/Contacts/b2CircleContact.cpp Box2D_v2.0.1/Box2Dfixed/Source/Dynamics/Contacts/b2CircleContact.cpp
--- Box2D_v2.0.1/Box2D/Source//Dynamics/Contacts/b2CircleContact.cpp	2008-04-10 23:37:08.000000000 -0700
+++ Box2D_v2.0.1/Box2Dfixed/Source/Dynamics/Contacts/b2CircleContact.cpp	2009-06-24 18:43:53.419350784 -0700
@@ -22,6 +22,7 @@
 #include "../../Common/b2BlockAllocator.h"
 
 #include <new>
+#include <cstring>
 
 b2Contact* b2CircleContact::Create(b2Shape* shape1, b2Shape* shape2, b2BlockAllocator* allocator)
 {
diff -rNuw Box2D_v2.0.1/Box2D/Source//Dynamics/Contacts/b2PolyAndCircleContact.cpp Box2D_v2.0.1/Box2Dfixed/Source/Dynamics/Contacts/b2PolyAndCircleContact.cpp
--- Box2D_v2.0.1/Box2D/Source//Dynamics/Contacts/b2PolyAndCircleContact.cpp	2008-04-12 14:29:14.000000000 -0700
+++ Box2D_v2.0.1/Box2Dfixed/Source/Dynamics/Contacts/b2PolyAndCircleContact.cpp	2009-06-24 18:44:14.947350642 -0700
@@ -22,6 +22,7 @@
 #include "../../Common/b2BlockAllocator.h"
 
 #include <new>
+#include <cstring>
 
 b2Contact* b2PolyAndCircleContact::Create(b2Shape* shape1, b2Shape* shape2, b2BlockAllocator* allocator)
 {
diff -rNuw Box2D_v2.0.1/Box2D/Source//Dynamics/Contacts/b2PolyContact.cpp Box2D_v2.0.1/Box2Dfixed/Source/Dynamics/Contacts/b2PolyContact.cpp
--- Box2D_v2.0.1/Box2D/Source//Dynamics/Contacts/b2PolyContact.cpp	2008-04-10 23:38:38.000000000 -0700
+++ Box2D_v2.0.1/Box2Dfixed/Source/Dynamics/Contacts/b2PolyContact.cpp	2009-06-24 18:43:31.433430087 -0700
@@ -23,6 +23,7 @@
 
 #include <memory>
 #include <new>
+#include <cstring>
 
 b2Contact* b2PolygonContact::Create(b2Shape* shape1, b2Shape* shape2, b2BlockAllocator* allocator)
 {
