--- Box2D_v2.0.1/Box2D/Source/Common/b2Settings.h	2008-04-10 23:44:06.000000000 +0200
+++ Box2D_v2.0.1/new/Source/Common/b2Settings.h	2010-07-11 00:14:22.144795808 +0200
@@ -72,7 +72,7 @@
 
 // Collision
 const int32 b2_maxManifoldPoints = 2;
-const int32 b2_maxPolygonVertices = 8;
+const int32 b2_maxPolygonVertices = 15;
 const int32 b2_maxProxies = 512;				// this must be a power of two
 const int32 b2_maxPairs = 8 * b2_maxProxies;	// this must be a power of two
 
