--- Box2D_v2.0.1/Box2D/Source/Dynamics/b2WorldCallbacks.h	2008-04-12 00:20:46.000000000 -0700
+++ Box2D_v2.0.1.new/Box2D/Source/Dynamics/b2WorldCallbacks.h	2009-06-27 09:48:46.409629367 -0700
@@ -92,18 +92,18 @@
 
 	/// Called when a contact point is added. This includes the geometry
 	/// and the forces.
-	virtual void Add(const b2ContactPoint* point) { B2_NOT_USED(point); }
+	virtual void Add(const b2ContactPoint* ) {  }
 
 	/// Called when a contact point persists. This includes the geometry
 	/// and the forces.
-	virtual void Persist(const b2ContactPoint* point) { B2_NOT_USED(point); }
+	virtual void Persist(const b2ContactPoint* ) { }
 
 	/// Called when a contact point is removed. This includes the last
 	/// computed geometry and forces.
-	virtual void Remove(const b2ContactPoint* point) { B2_NOT_USED(point); }
+	virtual void Remove(const b2ContactPoint* ) { }
 
 	/// Called after a contact point is solved.
-	virtual void Result(const b2ContactResult* point) { B2_NOT_USED(point); }
+	virtual void Result(const b2ContactResult* ) { }
 };
 
 /// Color for debug drawing. Each value has the range [0,1].
